amgut/lib/data_access/sql_connection.py

```diff
@@ -1,72 +1,204 @@
-from __future__ import division
-
 # -----------------------------------------------------------------------------
-# Copyright (c) 2014--, The American Gut Development Team.
+# Copyright (c) 2014--, The Qiita Development Team.
```

Comment:
I don't think this will get modified, just updated from qiita if necessary. I'll add info to contributing.md about it if wanted.